### DISCUSSION TITLE
2023-07-27

### DISCUSSION THREAD

**Discussion by woo-jk** [woowacourse-teams/2023-haru-study#170] [daily_meeting]
### 개인 일정
마코 - 2-3시 테코톡
노아 - 4-5시 근로

### 오늘의 할 일
- 소켓, 폴링 양방향 통신 어떻게 할지
- 정식 론칭 서비스 배포 완료

### 추가된 할 일
1. 재접속 안되는 문제 => 이슈파서 해결하기 서버 로직 문제 (백엔드는 해결됨, 프론트에 수정사항 전달하기)
2. 여러 개 스터디 참여 안되는 문제 => 트러블 슈팅해야함 아직 원인 파악 안됨(4번의 쿠키 관리 정책과 관련해서 같이 해결해야 함)
3. 빌드파일 배포 시 즉시 반영 안되는 문제 => 클라우드 플레어 캐싱 문제로 파악됨 (해결됨)
4. 쿠키 관리 정책 재논의

### 프론트엔드 리팩터링 목록

- fetch 
- 도메인 용어 및 문장 상수화
- 컴포넌트 폴더 구조
- 커스텀 훅 폴더 구조
- 라우터 및 api url 상수화
- 전체적인 코드 정리

### 언젠가 할 일
- 기획 변경된 이유 정리
- 질문 PlaceHolder로 유의미한 응답 예시를 제공하는 기능 (목표설정 가이드라인)
- 브랜치 전략 결정하기(dev -> main)

**Comment by yeopto** [comment]
### 체크인

5점 - 피곤합니다.

### 체크아웃

7점 - 배포가 기대되네요.

**Comment by woosung1223** [comment]
### 체크인
8점 - 너무 너무 덥네용

6점 - 제가 무슨 코드를 짰는지 기억이 안나요..

**Comment by nlom0218** [comment]
### 체크인

10점 - 어제 보다 배포가 더 잘 되겠죠..? 믿습니담

**Comment by woo-jk** [comment]
### 체크인
8점 - 조금 피곤합니다 ^_^

### 체크아웃
10점 - 리팩토링 잘해서 기분조음 👍

**Comment by aak2075** [comment]
### 체크인

10점 - 벌써 한 주가 끝나가네요 화이팅^^💪

### 체크아웃

10점 - 오늘도 작업을 잘 마무리 했네요👍

**Comment by MoonJeWoong** [comment]
### 체크인
10점 - 어제 잠을 일찍 자고 온데다 아침에 로깅 기능 이것저것 구현하면서 재미있었습니다. ㅎㅎ

### 체크아웃
8점 - 너무 많은 수정이 빠르게 일어나면서 서비스 로직이 잘 안그려지는 어려움이 있네요... 😭

**Comment by jaehee329** [comment]
### 체크인
7점 - 오늘이 벌써 목요일이라 -1 어제 배포 정상적으로 못 끝내서 -2

### 체크아웃
6점 - 피곤해졌어요
